src/_styles/components/_Toast.scss

```diff
@@ -0,0 +1,179 @@
+.toast {
+  display: flex;
+  align-items: center;
+  visibility: hidden;
+  max-width: 50px;
+  height: 50px;
+  /*margin-left: -125px;*/
```

Comment:
@Caleb remove this comment

Reply:
@NkFab Thanks for spotting this, I removed it!